### ISSUE TITLE
Retina Issue

### ISSUE THREAD

**Issue by nickff** [modxcms/imageSlim#12]
Hello,

First off, great plugin. 

Secondly, I'm looking to do something slightly different than what I.S. does normally, and wanted to get your thoughts. What I'm wanting is to be able to upload images at 2X their display size (3200px wide), keep that natural width on retina devices (though only displaying at 1600), but then reduce the natural width dynamically to 1600px for non-retina devices.

Basically, I'm trying to avoid having to make two versions of every image, one 2X for retina and one at display size. Is this possible with I.S? Let me know!
